server/boot/donate.js

```diff
@@ -62,6 +63,40 @@ export default function donateBoot(app, done) {
     });
   }
 
+  function onVerifyreCAPTCHA({ body, connection }, res, next) {
+
+    if (!body || !connection) {
+      return res.status(400).send({ error: 'Captcha validation failed' });
+    }
+
+    const { captchaResponse } = body;
+    const { remoteAddress } = connection;
+
+    if (!captchaResponse || !remoteAddress) {
+      return res.status(400).send({ error: 'Captcha validation failed' });
+    }
+
+    log('captchaResponse : ', captchaResponse);
+    log('remoteAddress   : ', remoteAddress);
+
+    const verificationURL =
+    'https://www.google.com/recaptcha/api/siteverify?secret=' +
+    keys.recaptcha +
+    '&response=' +
+    captchaResponse +
+    '&remoteip=' +
+    remoteAddress;
+
+    return request.post(verificationURL, function(error, response, body) {
```

Comment:
And params need to place in a request body:
```suggestion
    return request.post({
      url:verificationURL,
      form: {
        secret: keys.recaptcha,
        response: captchaResponse,
        remoteip: remoteAddress
      }
    }, function(error, response, body) {
```